Formula/dfu-util.rb

```diff
@@ -1,8 +1,8 @@
 class DfuUtil < Formula
   desc "USB programmer"
   homepage "https://dfu-util.sourceforge.io/"
-  url "https://downloads.sourceforge.net/project/dfu-util/dfu-util-0.9.tar.gz"
-  sha256 "36428c6a6cb3088cad5a3592933385253da5f29f2effa61518ee5991ea38f833"
+  url "https://downloads.sourceforge.net/project/dfu-util/dfu-util-0.10.tar.gz"
+  sha256 "a03dc58dfc79c056819c0544b2a5970537566460102b3d82cfb038c60e619b42"
   license "GPL-2.0"
```

Comment:
```suggestion
  license "GPL-2.0-or-later"
```

In `src/main.c`:

```c
/*
 * This program is free software; you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation; either version 2 of the License, or
 * (at your option) any later version.
 *
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with this program; if not, write to the Free Software
 * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 */
```